assets/admin/controllers.json

```diff
@@ -1,4 +1,43 @@
 {
-  "controllers": [],
+  "controllers": {
+    "@sylius/admin-bundle": {
```

Comment:
What's the reason to specify admin-bundle controllers inside appAdmin build? Aren't they already loaded in syliusAdmin build?

Reply:
To prevent double starting live controller when we sync package that update assets/controllers.json, yes, we don't start anymore stimulus in syliusAdmin build

Reply:
Can we define this list somewhere in sylius vendor and import from there?
So end-user that don't care for controllers won't need to manage that and who cares can override it.

My concern is sylius maintenence, this change make very easy to break stuff from end-user perspective.